talent/job-descriptions/infrastructure.md

```diff
@@ -0,0 +1,51 @@
+- "Delivering AI on Code" - source{d} is building the tech stack for large-scale machine learning on source code (MLoSC). We treat source code a first-class analyzable asset: by fetching and storing tens of millions of repositories from millions of developers around the world, processing and analyzing it with source{d} engine and Babelfish in order to build Machine Learning models. We make building blocks for the next generation of developer tools & systems that will change the way we learn, write & review code.
+
+- source{d} has raised a $6M round last summer and is rapidly growing team while preparing for the next round. We monetize by deploying our stack and applications at top tech companies. 
+
+## TEAM
+
+- Engineering department consists of five different teams, stacking up on each other:
+
+- Applications (Scala, Go and Python and Frontend tools), Builds CLI/Web applications based on ML research.
+- Machine Learning (C++ and Python), R&D of new ML models over the source code.
+- Data Retrieval (Scala and Go), Finds, fetches, stores and analyze over  +60M Git repositories.
+- Language Analysis (Go and another +15), Parses source code from different languages.
+- Infrastructure (Go), Manages on-prem bare metal servers with Kubernetes and CoreOS.
+
+
+## ROLE
+
+- The Infrastructure team focuses on building our computation cluster. This allows us to run large scale computation algorithms and store the required information for those computations.
+
+- Our cluster, with close to 2000 threads, more than 11TB of RAM and 1PB of storage, is backed by CoreOS and Kubernetes. It has two main goals: on the one hand, it stores all the available public code in a HDFS cluster and on the other hand, it runs intensive computation jobs over the stored data on top of Spark. 
+
+- Besides the main points above, the Infrastructure team is also in charge of several other important services, such as databases, queues, CI, monitorization, loging, etc.
+
+- At source{d}, we really care about open source,which is why we contribute to projects like Terraform & CoreOS, as well as creating our own projects across different teams at source{d}.
+
+- Good knowledge of containers, kubernetes and CoreOS is important. You will also be expected to have strong knowledge of Networking, Security, Linux (Kernel, Namespaces, cgroups). Finally, backend skills in at least one language and caring about best development practices, are required.
```

Comment:
I would not say that it is important the knowledge in containers, kubernetes and CoreOS. It dramatically narrows the amount of people that can fill the role.

The most important thing is strong knowledge in Linux. And Networking to some extent. The other one is one backend/scripting language

@tsolakoua If you need the exact phrase I can work on it :)